src/main/logs.ts

```diff
@@ -107,3 +107,17 @@ function createLairLogger(
     ),
   });
 }
+
+function identifierFromHolochainData(holochainData: HolochainData): string {
+  let identifier: string;
+  if (holochainData.version.type === 'built-in') {
+    identifier = `HOLOCHAIN ${holochainData.version.version} @ partition ${holochainData.partition}`;
+  } else if (holochainData.version.type === 'custom-path') {
+    identifier = `HOLOCHAIN CUSTOM BINARY @ partition ${holochainData.partition}`;
+  } else if (holochainData.version.type === 'running-external') {
+    identifier = `HOLOCHAIN EXTERNAL BINARY @ partition ${holochainData.partition}`;
+  } else {
+    identifier = `HOLOCHAIN unknown`;
+  }
+  return identifier;
+}
```

Comment:
Let's use early reutnr pattern insted of reading available

`function identifierFromHolochainData(holochainData) {
  if (holochainData.version.type === 'built-in') {
    return `HOLOCHAIN ${holochainData.version.version} @ partition ${holochainData.partition}`;
  }
  
  if (holochainData.version.type === 'custom-path') {
    return `HOLOCHAIN CUSTOM BINARY @ partition ${holochainData.partition}`;
  }
  
  if (holochainData.version.type === 'running-external') {
    return `HOLOCHAIN EXTERNAL BINARY @ partition ${holochainData.partition}`;
  }
  
  return `HOLOCHAIN unknown`;
}`